gateways/clients/responses/hathor_core/hathor_core_token_response.py

```diff
@@ -0,0 +1,35 @@
+from dataclasses import dataclass
+from typing import List, Optional
+from domain.tx.token import Token
+from dacite import from_dict
+
+
+@dataclass
+class HathorCoreTokenResponse:
+
+    name: Optional[str]
+    success: bool
+    message: Optional[str]
+    symbol: Optional[str]
+    total: Optional[int]
+    transactions_count: Optional[int]
+    melt: Optional[List[dict]]
+    mint: Optional[List[dict]]
+
+    @classmethod
+    def from_dict(cls, dikt: dict) -> 'HathorCoreTokenResponse':
+        return from_dict(data_class=cls, data=dikt)
+
+    def to_domain(self, id: str) -> Token:
```

Comment:
Didn't understand the name of this method

Reply:
`Token` is an object of domain (`domain.tx.token`). `HathorCoreTokenResponse` have all attributes based on what is expected from `hathor-core` response. So this method turns the response into a domain object

Reply:
I agree that `to_token` is better, so i changed.